### PR TITLE
 feat(project): #1214 username displayed on public project list

### LINKED ISSUE #1214: Public project list add owner

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #1214 

### Notes

A summary of what was achieved in this PR

- [ ] removed unused includes
- [ ] fixed html extra space
- [ ] project owner username displayed on public project list

Screenshot 2019-05-12 08 27 41

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Screenshot added

**Comment by eddiejaoude** [comment]
Oops forgot to add the new file 🤦‍♂ , updated last commit 🤓

**Comment by webkhushboo** [comment]
@eddiejaoude 
General questions: Do we need to show username on my projects page ? It will always be the owner for my projects case. 
Please see the attached screenshot and let me know your reviews on it.
image

**Comment by eddiejaoude** [comment]
Good question @webkhushboo !! 👍  I think it is ok, because I have only taken the first [owner](https://github.com/DashboardHub/PipelineDashboard/pull/1228/files#diff-9980ed0f0247b15217f41f9050070ee5R8) for now, where in the near future there will be multiple. So I think it is ok to appear on My Projects page.

**Comment by webkhushboo** [comment]
@eddiejaoude 
Okay thanks for clarifying. 👍

**Comment by eddiejaoude** [comment]
> imports in groups should follow alphabetical ordering

Good spot, I have updated the imports. 

Note: Comments about code are better inline with the code, because they can be dismissed after it is fixed, then the timeline doesn't get too busy 🤓

**Comment by webkhushboo** [comment]
Can we add single line comment before imports ?

**Comment by eddiejaoude** [comment]
> Can we add single line comment before imports ?

Sure, we need to keep the comments consistent throughout the application. What do you think of this?

```typescript
// 3rd party imports
import { Pipe, PipeTransform } from '@angular/core';
import { Observable } from 'rxjs';

// DashboardHub imports
import { UserModel } from '../shared/models/user.model';
import { UserService } from '../core/services/user.service';
```

**Comment by webkhushboo** [comment]
@eddiejaoude Sure !! I will do inline comments on code. I missed that option in review

**Comment by webkhushboo** [comment]
@angular/core seems to be Angular core components not third party.

**Comment by eddiejaoude** [comment]
> @angular/core seems to be Angular core components not third party.

I was thinking 3rd party to DashboardHub

File: web/src/app/pipes/user.pipe.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
+// 3rd party imports
```
[the remark this answers is not in the document]
Comment (eddiejaoude) [reply web/src/app/pipes/user.pipe.ts]: Please see comment from yesterday above https://github.com/DashboardHub/PipelineDashboard/pull/1228#issuecomment-491574864

**Pull request merged**